packages/migrate/commonsChunkPlugin/__snapshots__/commonsChunkPlugin.test.js.snap

```diff
@@ -0,0 +1,266 @@
+                        var foo = \\"bar\\";
+                        if (foo) {
+                            return /node_modulesfoo/.test(resource);
+                        } else if (foo === \\"something\\") {
+                            return /node_modulesbar/.test(resource);
+                        } else {
+                            return /node_modulesbaz/.test(resource);
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+"
+`;
+
+exports[`commonsChunkPlugin transforms correctly using "commonsChunkPlugin-7" data 1`] = `
+"module.export = {
+    optimizations: {
+        splitChunks: {
+            minChunks: Infinity,
+
+            cacheGroups: {
+                main: {
+                    name: 'main',
+                    chunks: 'initial',
+                    enforce: true
+                },
+
+                vendor: {
+                    name: 'vendor',
+                    chunks: 'initial',
+                    enforce: true,
+                    test: '/node_modules/'
+                }
+            }
+        }
+    }
+}
+"
+`;
+
+exports[`commonsChunkPlugin transforms correctly using "commonsChunkPlugin-8" data 1`] = `
+"module.export = {
+    optimizations: {
+        splitChunks: {
+            cacheGroups: {
+                main: {
+                    name: 'main',
+                    chunks: 'initial',
+                    enforce: true
+                },
+
+                vendor: {
+                    name: 'vendor',
+                    chunks: 'initial',
+                    enforce: true,
+                    test: '/node_modules/'
+                },
+
+                app: {
+                    name: 'app',
+                    chunks: 'initial',
+                    enforce: true
+                }
+            }
+        }
+    }
+}
+"
+`;
+
+exports[`commonsChunkPlugin transforms correctly using "commonsChunkPlugin-9" data 1`] = `
+"module.export = {
+    optimizations: {
+        splitChunks: {
+            cacheGroups: {
+                main: {
+                    name: 'main',
+                    chunks: 'initial',
+                    enforce: true
+                },
+
+                app: {
+                    name: 'app',
+                    chunks: 'initial',
+                    enforce: true
+                }
+            }
+        },
+
+        runtimeChunk: true
+    }
+}
+"
+`;
+
+exports[`commonsChunkPlugin transforms correctly using "commonsChunkPlugin-10" data 1`] = `
+"module.export = {
+    optimizations: {
+        splitChunks: {
+            minChunks: Infinity,
+
+            cacheGroups: {
+                modules: {
+                    name: 'modules',
+                    chunks: 'initial',
+                    enforce: true,
+                    test: ({ resource }) => /node_modules/.test(resource)
+                }
+            }
+        },
+
+        runtimeChunk: true
```

Comment:
This is tricky to figure out in multiple CCP instances. I think it's better to avoid multiple CCP cases and focus on getting single CCP case to be closer to SCP.